### ISSUE TITLE
"SyntaxError: Unexpected identifier" thrown if traced computed property contains multi-line comments

### ISSUE THREAD

**Issue by miluoshi** [mobxjs/mobx#1850]
The [trace](https://mobx.js.org/best/trace.html) function with last argument set to `true` enables to enter the debugger when re-evaluating traced derivation. If we trace a computed property with it's getter function body containing a multiline comment `/* */`, the browser throws `SyntaxError: Unexpected identifier` error. 

The [logTraceInfo](https://github.com/mobxjs/mobx/blob/0e6e0d24cbcb88e7970719770fc3dd263ade927d/src/core/observable.ts#L233) function is reponsible for injecting an anonymous function containing the debugger statement generated from a string. If we use `trace` to trace a computed value, the computed value's getter function body is converted into a string with [derivation.derivation.toString()](https://github.com/mobxjs/mobx/blob/0e6e0d24cbcb88e7970719770fc3dd263ade927d/src/core/observable.ts#L251) and then inserted into a comment of the generated function. By not escaping the generated string a multi-line comment is inserted as-is, thus early closing a comment in the generated function.

#### Example of generated string from the attached sandbox:
```
debugger;
/*
Tracing 'TestTrace@1.fullName'

You are entering this break point because derivation 'TestTrace@1.fullName' is being traced and 'TestTrace@1.firstName' is now forcing it to update.
Just follow the stacktrace you should now see in the devtools to see precisely what piece of your code is causing this update
The stackframe you are looking for is at least ~6-8 stack-frames up.

get fullName() {
        /* multi-line comment */ <---------------- early closed comment
        return `${this.firstName} ${this.lastName}`;
    }

The dependencies for this derivation are:

TestTrace@1.fullName
	TestTrace@1.firstName
	TestTrace@1.lastName
*/
```

#### Sandbox to reproduce the issue: 
https://codesandbox.io/s/v83p7xk80l

#### Suggested fix:
Escape (somehow) comments in the getter function's body string.
Inserting a space between closing comment tag (`* /`) should be enough. 
`derivation.derivation.toString()` -> `derivation.derivation.toString().replace(/\*\//g, '* /')`

**Comment by mweststrate** [comment]
Confirmed! Thanks for tracking this, I think we can fix this with a simple regex.

**Comment by mweststrate** [comment]
Fixed and released in 4.9.0 / 5.9.0
